Install/Test/CMakeLists.txt

```diff
@@ -0,0 +1,188 @@
+    target_link_libraries(v8 INTERFACE v8_libbase INTERFACE v8_libplatform)
+    target_include_directories(v8 INTERFACE "${V8_PACKAGE_PATH}/include")
+
+    set(V8_DIST
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/icudtl.dat"
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/icui18n.dll"
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/icuuc.dll"
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/v8.dll"
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/v8_libbase.dll"
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/v8_libplatform.dll"
+        "${V8_REDIST_PACKAGE_PATH}/lib/Release/zlib.dll")
+
+    # only 1 imported location per library -> Adding 1 library per file
+    foreach(V8FILE ${V8_DIST})
+        get_filename_component(V8FILE_NAME "${V8FILE}" NAME_WE)
+        add_library("v8::${V8FILE_NAME}" SHARED IMPORTED)
+        set_target_properties("v8::${V8FILE_NAME}" PROPERTIES IMPORTED_IMPLIB "${V8_PACKAGE_PATH}/lib/Release/v8_libbase.dll.lib")
+        set_target_properties("v8::${V8FILE_NAME}" PROPERTIES IMPORTED_LOCATION ${V8FILE})
+    endforeach()
+
+    set(EXTRA_LIBS
+        v8
+        v8inspector
+        llhttp
+        v8::icudtl
+        v8::icui18n
+        v8::icuuc
+        v8::v8
+        v8::v8_libbase
+        v8::v8_libplatform
+        v8::zlib)
+
+    if(CPU_ARCH STREQUAL "x64")
+        # Enable V8 Pointer Compression
+        # https://v8.dev/blog/pointer-compression
+        # https://stackoverflow.com/q/62921373
+        set(NAPI_DEFINITIONS PUBLIC V8_COMPRESS_POINTERS)
+    endif()
+endif()
+
+add_executable(TestInstall ${LOCAL_SCRIPTS} ${NPM_SCRIPTS} ${TEST_INSTALL_APP} ${HEADERS} ${JSI_CPP})
+
+# for napi
+target_compile_definitions(TestInstall PUBLIC NODE_ADDON_API_DISABLE_DEPRECATED)
+target_compile_definitions(TestInstall PUBLIC NODE_ADDON_API_DISABLE_NODE_SPECIFIC)
+
+set(INSTALL_LIBS_DIR "${BINARY_DIR}/install/lib")
+target_include_directories(TestInstall PRIVATE "${BINARY_DIR}/install/include")
+target_link_directories(TestInstall PRIVATE ${INSTALL_LIBS_DIR})
+
+
+target_link_directories(TestInstall PRIVATE "${V8JSI_LIB_PATH_RELEASE}")
+
+if(NAPI_JAVASCRIPT_ENGINE STREQUAL "JSI")
+    set(EXTRA_LIBS "v8jsi.dll.lib")
+endif()
+
+set(STD_LIBS AppRuntime
+    arcana
+    astc-encoder
+    bgfx
+    bimg
+    bx
+    Canvas
+    Console
+    edtaa3
+    etc1
+    etc2
+    ExternalTexture
+    GenericCodeGen
+    glslang
+    Graphics
+    iqa
+    JsRuntime
+    MachineIndependent
+    napi
+    NativeCamera
+    NativeCapture
+    NativeEngine
+    NativeInput
+    NativeOptimizations
+    NativeTracing
+    NativeXr
+    nvtt
+    OGLCompiler
+    openxr_loader
+    OSDependent
+    pvrtc
+    ScriptLoader
+    spirv-cross-core
+    spirv-cross-glsl
+    spirv-cross-hlsl
+    SPIRV
+    squish
+    tinyexr
+    UrlLib
+    Window
+    XMLHttpRequest
+    )
+
+target_link_libraries(TestInstall
+    ${STD_LIBS}
+    ${EXTRA_LIBS}
```

Comment:
We typically follow a pattern of using `ADDITIONAL_LINK_LIBRARIES` as the variable to add more libs. What does `STD_LIBS` mean here?

Reply:
`STD_LIBS` removed and `ADDITIONAL_LIBRARIES` name used